### ISSUE TITLE
Config hash mismatch with identical file

### ISSUE THREAD

**Issue by yobson1** [valheimPlus/ValheimPlus#84]
I directly uploaded the file from my local machine to the server using scp to ensure they are indentical and double checked using md5sum and it gives the same hash yet the server and client generate different hashes causing incompatible version errors when config enforcement is enabled.
This issue began occurring after updating to 0.8.5 from 0.8

**Comment by TheTerrasque** [comment]
What OS is on server, and what OS is on client?

**Comment by yobson1** [comment]
Ubuntu server, Windows client

**Comment by kachunkachunk** [comment]
A few of us are scratching our heads about this on the Discord server too. I've reproduced this on a fresh set of Linux dedicated server files, and fresh Windows client files, but generally can't be convincing enough that the server environment is up to the task.

I can disable all of the sections except the following, copy the config, validate the config file MD5 on both systems, and it will finally hash/match and allow a connection:

```
[Server]
; Change false to true to enable this section
enabled=true
; Modify the amount of players on your Server
maxPlayers=10
; Removes the requirement to have a server password
disableServerPassword=false
; This setting adds a version control check to verifiy that the server and yourself have the same configuration file installed.
enforceConfiguration=true
; This settings add a version control check to make sure that people that try to join your game or the server you try to join has V+ installed
enforceMod=true
; The total amount of data that the server and client can send per second in kilobyte
dataRate=60
; The interval in seconds that the game auto saves at (client only)
autoSaveInterval=1200
```

I see this on the Windows client end when it's loading some config sections, via the terminal window:
`[Error  : Unity Log]  Key NeedsServerSync not defined, using default value`
The Linux server says the same thing, usually during the same sections. But its logs also don't elaborate on what parameter is being parsed/loaded, so you'd have to walk through it slowly. I haven't exhaustively checked everything out between them, but in the case above, the autoSaveInterval seemed to possibly trigger this. If not, it's the Stamina section being loaded next (which is False. In fact all sections that follow are).

**Comment by nxPublic** [comment]
This is not a issue of V+

The dll for both, the server and the client are exactly identical. It is impossible to cause issues if you did everything correctly.

![image](https://user-images.githubusercontent.com/39051287/108832637-aa65e400-7580-11eb-9c51-ab4f466c9359.png)

**Comment by nxPublic** [comment]
We have resolved this issue and uploaded silently all released packages of 0.8.5.
